modules/siddhi-core/src/main/java/org/wso2/siddhi/core/util/snapshot/state/SnapshotStateList.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.siddhi.core.util.snapshot.state;
+
+import java.io.Serializable;
+import java.util.TreeMap;
+
+/**
+ * The class which encloses multiple snapshot states.
+ */
+public class SnapshotStateList implements Serializable {
+    private TreeMap<Long, SnapshotState> snapshotStates = new TreeMap<>();
```

Comment:
Don't we need a serialVersionUUID?

Reply:
Looks like it's not needed here as the find bugs plugin did not complain on this.